addons/token-exchange/token_exchanger_agent.go

```diff
@@ -6,27 +6,24 @@ import (
 	"time"
 
 	"github.com/openshift/library-go/pkg/controller/controllercmd"
-	"github.com/openshift/library-go/pkg/controller/factory"
 	"github.com/openshift/library-go/pkg/operator/events"
 	"github.com/openshift/library-go/pkg/operator/resource/resourceapply"
 	"github.com/spf13/cobra"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/version"
 	"k8s.io/client-go/informers"
-	corev1informers "k8s.io/client-go/informers/core/v1"
 	"k8s.io/client-go/kubernetes"
 	corev1lister "k8s.io/client-go/listers/core/v1"
-	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/klog/v2"
 	"open-cluster-management.io/addon-framework/pkg/lease"
 )
 
 const (
-	TokenExchangeName = "tokenexchange"
+	TokenExchangeName   = "tokenexchange"
+	CreatedByLabelKey   = "multicluster.odf.openshift.io/created-by"
+	CreatedByLabelValue = "token-exchange-agent"
```

Comment:
better to use `TokenExchangeName` value here. makes it easier to backtrack.

Reply:
This is coming from a dependent PR. Let us take it up as part of cleanup.